packages/actor-tests/bedrock/account-sends.ts

```diff
@@ -0,0 +1,34 @@
+import { Wallet, providers } from 'ethers'
+import { expect } from 'chai'
+
+import { actor, setupActor, run } from '../lib/convenience'
+import { devWalletsL2 } from './utils'
+
+interface Context {
+  wallet: Wallet
+}
+
+actor('Sender', () => {
+  let provider: providers.JsonRpcProvider
+
+  let wallets: Wallet[]
+
+  setupActor(async () => {
+    provider = new providers.JsonRpcProvider(process.env.L2_RPC)
+    wallets = devWalletsL2()
+  })
+
+  run(async (b, ctx: Context, logger) => {
+    const sender = wallets[Math.floor(Math.random() * wallets.length)]
+    const recipient = Wallet.createRandom().connect(provider)
```

Comment:
Do we want to send the funds to one of the wallets so we cycle the ETH instead of losing it forever?

Reply:
This test uses the dev accounts in state, which have a gigantic amount of eth. The test itself only sends 66 wei, so  don't think this will be an issue. I'll make sure eth is recoverable once we use non-dev accounts.

Reply:
Fixed to use recoverable account.